config/config.go

```diff
@@ -1025,7 +1025,18 @@ type Config struct {
 		Vault  VaultConfig  `json:"vault"`
 	} `json:"kv"`
 
-	// Secrets are key-value pairs that can be accessed in the dashboard via "secrets://"
+	// The TYK_GW_SECRETS environment variable allows you to configure the "secrets"
+	// section in the Tyk Gateway configuration file. These secrets are key-value pairs
+	// that can be dynamically injected into API Definitions and the tyk.conf configuration
```

Comment:
Can we provide an example of a secret being used to dynamically inject a value into tyk.conf?

Reply:
@rewsmith I have updated with an example

Reply:
@rewsmith Can you review this again? I'll be taking the responsibility of this PR now.